examples/cis3-nft-sponsored-txs/src/lib.rs

```diff
@@ -905,35 +814,30 @@ fn contract_permit<S: HasStateApi>(
     // Parse the parameter.
     let param: PermitParam = ctx.parameter_cursor().get()?;
 
-    let message = param.message;
-
     ensure!(
-        param.signature.len() == 1
-            && param.signature.get(&0).ok_or(CustomContractError::SignatureMapMisconfigured)?.len()
+        param.signature.sigs.len() == 1
+            && param
+                .signature
+                .sigs
+                .get(&0)
+                .ok_or(CustomContractError::SignatureMapMisconfigured)?
+                .sigs
+                .len()
```

Comment:
Consider introducing a variable for this and use the variable name to help the reader understand what this is

Reply:
This check is for making sure that the signature is at index 0 in both signature maps. Since we now support multi-sig accounts, this check was removed altogether.